pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1836,14 +1836,23 @@ func handleGlobalConfigDelete(ctxArg interface{}, key string,
 
 // getCloudInitUserData : returns decrypted cloud-init user data
 func getCloudInitUserData(ctx *domainContext,
-	config types.DomainConfig) (*string, error) {
-	status, clearTextData, err := utils.GetCipherData(agentName,
-		config.CipherBlockStatus, config.CloudInitUserData)
-	publishCipherBlockStatus(ctx, status)
-	if status.IsCipher && len(status.Error) == 0 {
-		log.Infof("%s, cipherblock decryption successful\n", config.Key())
-	}
-	return clearTextData, err
+	dc types.DomainConfig) (zconfig.EncryptionBlock, error) {
+	if dc.CipherBlockStatus.IsCipher {
+		status, decBlock, err := utils.GetCipherCredentials(agentName,
+			dc.CipherBlockStatus)
+		ctx.pubCipherBlockStatus.Publish(status.Key(), status)
+		if err != nil {
+			log.Infof("%s, domain config cipherblock decryption unsuccessful: %v\n", dc.Key(), err)
```

Comment:
It would make sense to make these log.Error for all three decryption failures, and also add "falling back to cleartext" to the log message. That makes it easier to find it in Kibana.

Reply:
Makes sense. I will update it.